kernelci/cli/base.py

```diff
@@ -533,6 +537,13 @@ def __init__(self, path, command, cli_args, section=None):
                   way to have default values for each CLI tool
         """
         self._deprecated_settings = False
+        if path is None and os.environ.get('KCI_SETTINGS'):
+            path = os.environ.get('KCI_SETTINGS')
+            if not os.path.exists(path):
+                print(f"Error: KCI_SETTINGS points to a non-existing "
+                      f"file: {path}", file=sys.stderr)
+                sys.exit(1)
```

Comment:
Arguably this should be throwing an exception, exiting in a class constructor is rather unexpected I think.  The logic is fine though, as the program should print this error and abort.  Maybe for services that aren't CLI tools it would need to handle the error differently etc.

Reply:
Maybe better i check after everything path variable and make sure file exist? And if doesnt i can throw exception

Reply:
Yes, that might be better as a separate commit.  I think right now there will be an exception anyway when trying to open a file that doesn't exist.

Reply:
Done, please check if it is better now